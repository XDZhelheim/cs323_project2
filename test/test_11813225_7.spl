int super_test()
{
  int x;
  undefined(1, 2);
  1=x;
}

char super_test()
{
  int cui = -1;
}