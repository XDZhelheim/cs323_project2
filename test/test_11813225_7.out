Error type 2 at Line 4: a function is invoked without a definition
Error type 6 at Line 5: rvalue appears on the left-hand side of the assignment operator
Error type 4 at Line 8: a function is redefined
